.github/workflows/main.yml

```diff
@@ -14,31 +14,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.8
+          python-version: '3.8'
       - name: Install dependencies
         run: python -m pip install tox
       - name: Run linting
         run: python -m tox -e lint
 
-  types:
-    strategy:
-      matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python }}
-      - name: Install dependencies
-        run: python -m pip install tox
-      - name: Run type-checking
-        run: python -m tox -e types
-
   test:
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: ['3.6', '3.7', '3.8', '3.9', '3.10']
```

Comment:
I've started storing these changes line by line ([example](https://github.com/jaraco/skeleton/blob/aae281a9ff6c9a1fa9daad82c79457e8770a1c7e/.github/workflows/main.yml#L9-L12)) as I've found it makes the diffs easier to read. Same for `platform`.

Reply:
Good call. Done in https://github.com/pypa/twine/pull/827/commits/0a456e12aae5e57dd46e03942a15a7c670c7b392.